browse-scaler/src/index.js

```diff
@@ -30,21 +30,24 @@ const buildResponse = image => {
  * 'dataset' refers to collections
  * @returns {String} image url or null
  */
-const getImageUrlFromConcept = async (conceptId, conceptType) => {
+const getImageUrlFromConcept = async (conceptId, conceptType, imageSrc) => {
```

Comment:
update doc-string

Reply:
This is actually being taken from the event path so we can still reduce the passing of the variable but, we should rethink the event path in that case. This is captured in a separate epic.